index.html

```diff
@@ -71,7 +71,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
                             <h3>Monero is fungible.</h3>
-                            <p>Because of the on-by-default nature of the privacy technologies, Monero is fungible, meaning one Monero will always be equal to another. This means you won't have to worry about Monero blacklisted by exchanges or vendors.</p>
+                            <p>Privacy is always-on. Therefore Monero is fungible, meaning  Monero will never be blacklisted by exchanges or vendors due to its association in previous transactions.</p>
```

Comment:
We can't use the word 'never'. It was in the original copy, but was dropped because it's an absolute statement that puts a lot of trouble to back up. And I think a slightly longer explanation of what 'fungibility' is (like in the original) is warranted. Again, just my opinion.

Great work! Thanks for working on cleaning this up.